app/src/main/java/com/globalaccelerex/mpossdksample/viewModel/ItemViewModel.kt

```diff
@@ -26,9 +27,11 @@ class ItemViewModel : ViewModel() {
         }
     }
 
-    val totalPrice = Transformations.map(selectedItemList) { checkOutItemList ->
+    private val _totalPrice = Transformations.map(selectedItemList) { checkOutItemList ->
```

Comment:
You don't need to make totalPrice Mutable. To reset it, simply clear your selectedItemList

Reply:
Has been resolved Sir